<launch>
   <node pkg="nmpc_coverage_control" name="heterogeneous_system_formation_sim" type="heterogeneous_system_formation_sim" output="screen" >
     <param name="KPx" value="1" />
     <param name="KPy" value="1" />
     <param name="KPz" value="1" />
	 <param name="KP_yaw" value="1" />
	 <param name="K_Formation_x" value="3.0" />
     <param name="K_Formation_y" value="3.0" />
     <param name="weight_target" value="1.5" />
     <param name="gama_1" value="0.5" />
	 <param name="desired_heading_1" value="0.314" />
     <param name="desired_heading_2" value="5.69" />
     <param name="desired_heading_3" value="2.862" />
     <param name="dis_1_x" value="-6" />
     <param name="dis_1_y" value="0" />
     <param name="dis_1_z" value="3" />
     <param name="dis_2_x" value="-4.974" />
     <param name="dis_2_y" value="3.355" />
     <param name="dis_2_z" value="1" />
     <param name="dis_3_x" value="5.768" />
     <param name="dis_3_y" value="-1.654" />
     <param name="dis_3_z" value="1" />
   </node>
   <node pkg="nmpc_coverage_control" name="heterogeneous_system_ukf_sim" type="heterogeneous_system_ukf_sim" output="screen" >
     <param name="topic_box_uav1" value="/uav1/YOLO/box" />
     <param name="P_init_0" value="0.001" />
     <param name="P_init_1" value="0.001" />
     <param name="P_init_2" value="0.001" />
	 <param name="P_init_3" value="0.001" />
	 <param name="measurement_noise_0" value="4" />
     <param name="measurement_noise_1" value="4" />
     <param name="measurement_noise_2" value="0.0009" />
	 <param name="measurement_noise_3" value="0.0001" />
	 <param name="process_noise_0" value="0.0001" />
     <param name="process_noise_1" value="0.0001" />
     <param name="process_noise_2" value="0.0001" />
	 <param name="process_noise_3" value="0.0001" />
	 <param name="x_init_0" value="1.0" />
	 <param name="x_init_1" value="1.0" />
	 <param name="x_init_2" value="0.5" />
	 <param name="x_init_3" value="0.0" />
   </node>
</launch>
